packages/nx/src/utils/assert-workspace-validity.ts

```diff
@@ -19,44 +16,16 @@ export function assertWorkspaceValidity(
 
   const invalidImplicitDependencies = new Map<string, string[]>();
```

Comment:
We don't need this map right?

Reply:
We do need it for the project level `implicitDependencies` below (line 59).